### ISSUE TITLE
list-devices doesn't work on Windows

### ISSUE THREAD

**Issue by yiguanxianyu** [doronz88/pymobiledevice3#263]
**Test environment**
- Windows 11 Pro 22621.1
- iPhone 12 mini & iOS15.5

**Describe the bug**
`list-devices`  doesn't work on v1.21.0 or later, but works on v1.20.6 or earlier.

**To Reproduce**
`$ pymobiledevice3 list-devices`

**Expected behavior**
Show [{*Device Info*}]

**Logs**
```
$ pymobiledevice3 list-devices          
       
Traceback (most recent call last):
  File "C:\dev\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\dev\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "PROJECT_VENV\Scripts\pymobiledevice3.exe\__main__.py", line 7, in <module>
  File "PROJECT_VENV\lib\site-packages\pymobiledevice3\__main__.py", line 51, in cli
    cli_commands()
  File "PROJECT_VENV\lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "PROJECT_VENV\lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "PROJECT_VENV\lib\site-packages\click\core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "PROJECT_VENV\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "PROJECT_VENV\lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "PROJECT_VENV\lib\site-packages\pymobiledevice3\cli\list_devices.py", line 20, in list_devices
    for device in usbmux.list_devices():
  File "PROJECT_VENV\lib\site-packages\pymobiledevice3\usbmux.py", line 367, in list_devices
    mux = create_mux()
  File "PROJECT_VENV\lib\site-packages\pymobiledevice3\usbmux.py", line 363, in create_mux
    return MuxConnection.create()
  File "PROJECT_VENV\lib\site-packages\pymobiledevice3\usbmux.py", line 146, in create
    sock = MuxConnection.create_usbmux_socket()
  File "PROJECT_VENV\lib\site-packages\pymobiledevice3\usbmux.py", line 139, in create_usbmux_socket
    return SafeStreamSocket(MuxConnection.ITUNES_HOST, socket.SOCK_STREAM)
  File "PROJECT_VENV\lib\site-packages\pymobiledevice3\usbmux.py", line 101, in __init__
    self.sock.connect(address)
OSError: connect(): bad family
```

**Comment by doronz88** [comment]
Thanks for creating the issue! @yiguanxianyu 
I created a fix and published a new version [v1.22.1](https://github.com/doronz88/pymobiledevice3/releases/tag/v1.22.1). 
Can you verify the issue is now resolved?

**Comment by yiguanxianyu** [comment]
It works fine now, thanks!
